integrations/cli/index.test.ts

```diff
@@ -52,9 +52,14 @@ describe.each([
             addVariant('hocus', ['&:focus', '&:hover'])
           }
         `,
+        'project-a/tailwind.config.js': js`
+          module.exports = {
+            content: ['../project-b/src/**/*.js'],
+          }
+        `,
         'project-a/src/index.css': css`
           @import 'tailwindcss/utilities';
-          @source '../../project-b/src/**/*.js';
+          @config '../tailwind.config.js';
```

Comment:
Hmmm. I was initially thinking no but since we do return `origin` of `undefined` for globs defined inside the css file, it does behave differently (and that is implemented in each client right now). I wanted to avoid having to split all tests again to support that though, perhaps we can at least include both cases in the same test (so it doesn't mean we double all our basic integration tests right away).